store/cachemulti/store.go

```diff
@@ -124,7 +104,7 @@ func newCacheMultiStoreFromCMS(cms Store) Store {
 		stores[k] = v
 	}
 
-	return NewFromKVStore(cms.db, stores, nil, cms.traceWriter, cms.traceContext)
+	return NewFromKVStore(cms.db, stores, nil, cms.traceWriter, cms.traceContext, cms.locks)
```

Comment:
Documenting discussion from online: we intentionally pass around pointer to `sync.Map` so cached contexts respect the same locks.